spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -40,15 +44,19 @@
   context 'autocomplete results' do
     it 'returns collection of matches' do
       create(:version, :production)
-      7.times { create(:institution, :contains_harv) }
+      7.times { create(:institution, :contains_harv, approved: true) }
+      # adding a non approved institutions row
+      create(:institution, :contains_harv, approved: false)
```

Comment:
You can remove lines 48 and 49.  We are testing autocomplete of `approved` records and having this non-approved record does not affect it.